src/components/Plan.js

```diff
@@ -73,12 +123,26 @@ class Plan extends Component {
                 longitude: station.longitude,
               }}
               image = { this.getStationIcon(station) }
-              title={`${station.extra.internal_id} ${station.name}`}
-              description={`${station.free_bikes} bikes | ${station.empty_slots} empty slots`}
               key={station.id}
               ref={station.id}
               onPress={this.onMarkerPress.bind(this)}
               >
+
+              <MapView.Callout onPress={() => this.onFavPress(station)}>
```

Comment:
I like your idea of introducing Callout. However have you tested it on Android Virtual Device or even better on actual device. Is the map being rendered smoothly without any glitches?
Cause that's why I havn't used them initially

Reply:
I tested it on my old samsung galaxy s4 and it looks normal to me.